### PR TITLE
Add missing API methods

### PR DESCRIPTION
**Pull request by megazoll**
https://developer.uber.com/v1/api-reference/

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -237,6 +237,21 @@ public function getPromotions($attributes = [])
...
 
     /**
+     * The Request Estimate endpoint allows a ride to be estimated given the
+     * desired product, start, and end locations. If the end location is
+     * not provided, only the pickup ETA and details of surge pricing
+     * information are provided. If the pickup ETA is null, there are no cars
+     * available, but an estimate may still be given to the user.
+     *
+     * @param    array    $attributes   Query attributes
+     * @return   stdClass               The JSON response from the request
```
Comment (stevenmaguire) [inline src/Client.php:248]: Could you please include an extra line between @params and @return?

**Comment by stevenmaguire** [comment]
Could you please update the README with documentation on these new end points?

**Comment by megazoll** [comment]
@stevenmaguire updated

**Comment by stevenmaguire** [comment]
Great Thank you.

**Pull request merged**